ios/brave-ios/Sources/Brave/Frontend/Browser/InfoBar.swift

```diff
@@ -0,0 +1,141 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import DesignSystem
+import Foundation
+import Shared
+import SnapKit
+import SwiftUI
+import UIKit
+
+private struct InfoBarUX {
+  static let toastPadding: CGFloat = 10.0
+  static let toastCloseButtonWidth: CGFloat = 20.0
+  static let toastLabelFont = UIFont.preferredFont(forTextStyle: .subheadline)
+  static let toastBackgroundColor = UIColor(braveSystemName: .schemesOnPrimaryFixed)
+}
+
+class InfoBar: Toast, UITextViewDelegate {
```

Comment:
nit: Infobar

Reply:
Discussed in DM and decided to keep it as it is, because chromium uses `InfoBar`